tasks/main.yaml

```diff
@@ -17,14 +30,23 @@
         - item.0.type|d('') == 'elasticsearch'
         - item.1.state|d('present') == 'present'
 
+    - name: Set rsyslog_files_and_forwards
+      set_fact:
+        rsyslog_files_and_forwards: "{{ rsyslog_files_and_forwards|d([]) }} + {{ [  item.0|combine( {'logs_collections_name': item.1.name} )] }}"
+      with_subelements:
+        - "{{ logging_outputs }}"
+        - logs_collections
+      when:
+        - item.1.state|d('present') == 'present'
```

Comment:
@nhosoi I believe you are missing a when condition here.
- item.0.type|d('') == 'files_forwards'